benchmarks/templates/benchmarks/tutorials/tutorial-info-section.html

```diff
@@ -2,7 +2,7 @@
     <div class="title">Stuck?</div>
     <div class="content">
         <p class="benefits_info is-size-5-mobile">
-            Our tutorial was made in conjunction with Brain-Score’s users and we hope it covers everything.
+            Our tutorials and FAQs were made in conjunction with Brain-Score’s users and we hope it covers everything.
             Sometimes though, things come up, and users can get stuck. If that is the case, please reach out to the
```

Comment:
```suggestion
            Sometimes though, things come up, and users can get stuck. If that is the case, please reach out to the
```
how can they reach out? Recommend linking the community page here, or Github directly

Reply:
Community page is linked below via button underneath the first section